interfaces/IBF-dashboard/src/app/components/chat/chat.component.ts

```diff
@@ -187,4 +192,52 @@ export class ChatComponent implements OnDestroy {
 
     await alert.present();
   }
+
+  public async closePcodeEventPopup(area) {
+    const message = this.closeEventPopup['message'].replace(
+      '{{ name of district }}',
+      area.name,
+    );
+    const alert = await this.alertController.create({
+      message: message,
+      buttons: [
+        {
+          text: this.closeEventPopup['cancel'],
+          handler: () => {
+            console.log('Cancel close pcode');
+          },
+        },
+        {
+          text: this.closeEventPopup['confirm'],
+          handler: () => {
+            this.closePcodeEvent(area.id, area.pcode);
+          },
+        },
+      ],
+    });
+    await alert.present();
+  }
+
+  public async closePcodeEvent(eventPcodeId: number, pcode: string) {
+    this.apiService.closeEventPcode(eventPcodeId);
+    this.filteredAreas = this.filteredAreas.filter(
+      (item) => item.id !== eventPcodeId,
+    );
+    this.countryService
+      .getCountrySubscription()
+      .subscribe((country: Country) => {
+        this.analyticsService.logEvent(AnalyticsEvent.watchVideoGuide, {
+          page: AnalyticsPage.dashboard,
+          country: country.countryCodeISO3,
+          isActiveEvent: this.eventService.state.activeEvent,
+          isActiveTrigger: this.eventService.state.activeTrigger,
+          pcode: pcode,
+        });
+      });
+    let loading = await this.loadingCtrl.create({});
```

Comment:
The goal for this loader is just to give users an impression that something happens. The API call `closePcodeEvent` api call is very fast and otherwise you do not notice that one of the pcode area disappears from the list

Reply:
I thought we had a result popup that explicitly informed the user of the update.

Reply:
Nope, that's not there and its not in the refinement. So i suggest to leave it to finish this items